decompose con1d to conv2d for cuda backend

Gasoonjia opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #15017

this diff decomposed conv1d into conv2d for aoti-cuda backend support

Differential Revision: [D84296877](https://our.internmc.facebook.com/intern/diff/D84296877/)

Gasoonjia committed:
decompose con1d to conv2d for cuda backend

this diff decomposed conv1d into conv2d for aoti-cuda backend support

Differential Revision: [D84296877](https://our.internmc.facebook.com/intern/diff/D84296877/)

[ghstack-poisoned]

larryliu0820 reviewed (approved):
Review automatically exported from Phabricator review in Meta.

Gasoonjia committed:
Update on "decompose con1d to conv2d for cuda backend"


this diff decomposed conv1d into conv2d for aoti-cuda backend support

Differential Revision: [D84296877](https://our.internmc.facebook.com/intern/diff/D84296877/)

[ghstack-poisoned]

Gasoonjia committed:
Update on "decompose con1d to conv2d for cuda backend"


this diff decomposed conv1d into conv2d for aoti-cuda backend support

Differential Revision: [D84296877](https://our.internmc.facebook.com/intern/diff/D84296877/)

[ghstack-poisoned]

Gasoonjia committed:
Update on "decompose con1d to conv2d for cuda backend"


this diff decomposed conv1d into conv2d for aoti-cuda backend support

Differential Revision: [D84296877](https://our.internmc.facebook.com/intern/diff/D84296877/)

[ghstack-poisoned]

Gasoonjia committed:
Update on "decompose con1d to conv2d for cuda backend"


this diff decomposed conv1d into conv2d for aoti-cuda backend support

Differential Revision: [D84296877](https://our.internmc.facebook.com/intern/diff/D84296877/)

[ghstack-poisoned]
